cross-platform/react-app/src/frontend/CameraSample/CameraSampleApp.tsx

```diff
@@ -24,15 +32,20 @@ function ImageSelectionHandler() {
 }
 
 export function CameraSampleApp() {
-  return <App getModelScreenExtensions={(iModel: IModelConnection): ModelScreenExtensionProps => {
-    return {
-      toolsBottomPanel: CameraSampleToolsBottomPanel,
-      additionalComponents: <ImageSelectionHandler />,
-      additionalTabs: [{
-        label: i18n("PicturesBottomPanel", "Pictures"),
-        isTab: true,
-        popup: <PicturesBottomPanel key="pictures" iModel={iModel} />,
-      }],
-    };
-  }} />;
+  return <App
+    onInitialize={async () => {
+      await IModelApp.localization.registerNamespace("CameraSampleApp");
```

Comment:
Does our existing code have a convention for doing await like this on the last line of an async function vs just returning the result? If so, and we return the result instead of using await, you should replace the `await` here with `return`.

Reply:
Not that I could find. Is returning the Promise considered better form?

Reply:
I honestly don't know, so if we don't do so consistently already, then what you have is fine.